test/e2e/flask/create-watch-account.spec.ts

```diff
@@ -5,7 +5,7 @@ import FixtureBuilder from '../fixture-builder';
 import { defaultGanacheOptions, unlockWallet, withFixtures } from '../helpers';
 import { Driver } from '../webdriver/driver';
 
-const ACCOUNT_1 = '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1';
+const ACCOUNT_1 = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
```

Comment:
This shouldn't be needed anymore, see [here](https://github.com/MetaMask/metamask-extension/blob/e9da8e3f808530907a73f0a1b940d5abb0bf608e/test/e2e/flask/create-watch-account.spec.ts#L167-L170):

```suggestion
const ACCOUNT_1 = '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1';
```